nextcloudappstore/core/user/forms.py

```diff
@@ -1,11 +1,11 @@
-from registration.forms import RegistrationFormUniqueEmail
+from django import forms
+
 from captcha.fields import ReCaptchaField
 
 
-class RegistrationFormRecaptcha(RegistrationFormUniqueEmail):
+class SignupFormRecaptcha(forms.Form):
     """integrate a recaptcha field."""
     recaptcha = ReCaptchaField()
 
-    def clean(self):
-        self._validate_unique = True
-        super().clean()
+    def signup(self, request, user):
```

Comment:
We need it becuase allauth assume that if you create your own signup form, then you need an extra hook to do what needs doing when you sign up a user. I think the social signup view uses this method to acutally create the django user and hook the social token to the user. If the method isn't defined allauth raises an exception.

I guess I could make a no-op out of it and just you pass instead of return None.